### PR TITLE
[Add] Readme for set mongodb on kube

### PR DESCRIPTION
**Pull request by A-Phat**

### CODE REVIEW

File: kubernate/resource/documents/Readme.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+ # <img src="../images/mongodb.png" width="" height="30"> Mongodb on <img src="../images/kubernetes.png" width="" height="27"> Kubernetes
+
+## Create Cluster by KinD
```
Comment (rosered11) [inline kubernate/resource/documents/Readme.md:3]: Create CLuster by Kind should that "Create cluster by KinD"

Comment (rosered11) [inline kubernate/resource/documents/Readme.md]: This line should have `Create cluster by KinD`

**Review by rosered11 — CHANGES_REQUESTED** [review]

File: kubernate/resource/documents/Readme.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+ # <img src="../images/mongodb.png" width="" height="30"> Mongodb on <img src="../images/kubernetes.png" width="" height="27"> Kubernetes
+
+## Create cluster by KinD
+***[KinD](https://kubernetes.io/blog/2020/05/21/wsl-docker-kubernetes-on-the-windows-desktop/): Kubernetes made easy in a container***. 
+<br>
+<br>
+Right now, we have Docker that is installed, configured and the last test worked fine.
+
+This is normal as we didn't enable the Docker Kubernetes cluster. So let's install KinD and create our first cluster.
```
Comment (rosered11) [inline kubernate/resource/documents/Readme.md:9]: Can I use Minikube instead the Kind?

File: kubernate/resource/documents/Readme.md (hunk 1 above)
Comment (A-Phat) [reply kubernate/resource/documents/Readme.md]: I fixed it.

File: kubernate/resource/documents/Readme.md (hunk 2 above)
Comment (A-Phat) [reply kubernate/resource/documents/Readme.md:9]: I don't know. I haven't studied it much.

**Review by rosered11 — APPROVED** [review]

**Pull request merged**